src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -0,0 +1,127 @@
+      </div>
+      <div className="tabs-container">
+        <div className="container">
+          <ul className="nav nav-tabs">
+            <li className={`nav-item ${
+                selectedTabIndex == 0 ? 'active' : ''
+              }`} aria-selected={selectedTabIndex == 0}>
+              <button onClick={() => setSelectedTabIndex(0)}>
+                Access Actions Required
+              </button>
+            </li>
+            <li className={`nav-item ${
+                selectedTabIndex == 1 ? 'active' : ''
+              }`} aria-selected={selectedTabIndex == 1}>
+              <button onClick={() => setSelectedTabIndex(1)}>
+                Download List
+                {!isError && !isFetching && fileStats &&
+                  <span className="fileCount">{fileStats.numberOfFilesAvailableForDownload}</span>
+                }
+              </button>
+            </li>
+          </ul>
+        </div>
+      </div>
+      {selectedTabIndex == 0 && !isError && !isFetching && fileStats &&
+        <div>
+          {fileStats.numberOfFilesRequiringAction > 0 && 
+            <div>
+              <div className="subSectionOverview">
+                <div className="container">
+                  <div className="subSectionContainer">
+                    <span className="subSectionTitle">You Have Files Which Require Actions Before Downloading</span>
+                  </div>
+                  <p className="description">The items in this category require different actions in order 
+                  to download them. Select Start to complete the action.</p>
+                </div>
+              </div>
+              <div className="actionsRequiredContainer container">
+                <DownloadListActionsRequired />
+              </div>
+            </div>}
+            {fileStats.numberOfFilesRequiringAction === 0 && <div className="placeholder">
+                <div>No actions are currently required.</div>
+              </div>}
+        </div>
+      }
+      {selectedTabIndex == 1 && !isError && !isFetching && fileStats &&
+        <div>
+          {fileStats.numberOfFilesAvailableForDownload > 0 && 
+            <div>
+              <div className="subSectionOverview">
+                <div className="container">
+                  <div className="subSectionContainer">
+                    <span className="subSectionTitle">Complete Your Download</span>
+                    <DownloadListStats />
+                  </div>
+                  <p className="description">Downloading your files programmatically is the quickest and most efficient way to get all of your files, 
+                  both internal and externally hosted. Metadata will always be included in your download automatically when downloading programmatically. 
+                  If you choose to download as .zip files, you can download external files individually at any time.</p>
```

Comment:
Not code related, but this copy is confusing to me and I think this needs another look before this is surfaced.

Reply:
Agreed.  I think it was done in haste in the design mockup, but I consider it a work in progress.